### PR TITLE
feat(launch): adding `agent.nodeSelector` variable for agent chart 

### PR DESCRIPTION
**Pull request by bcsherma**
Adding a variable for the agent pod's `nodeSelector` in the values file for the agent chart.

### CODE REVIEW

File: charts/launch-agent/README.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ The table below describes all the available variables in the chart:
 | `agent.image`                  | string          | No                   | `wandb/launch-agent-dev:latest` | Container image for the agent.                                                                                                                   |
 | `agent.imagePullPolicy`        | string          | No                   | `Always`                        | Pull policy for the agent container image.                                                                                                       |
 | `agent.resources`              | object          | No                   | Limit to 1 CPU, 1Gi RAM         | Pod spec resources block for the agent.                                                                                                          |
+| `agent.nodeSelector`           | object          | No                   | `{}`                            | Node selector for the agent pod.                                                                                                                 |
```
Comment (KyleGoyette) [inline charts/launch-agent/README.md:49]: Does leaving this blank break anything?

File: charts/launch-agent/Chart.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ name: launch-agent
 icon: https://em-content.zobj.net/thumbs/240/apple/354/rocket_1f680.png
 description: A Helm chart for running the W&B Launch Agent in Kubernetes
 type: application
-version: 0.11.0
+version: 0.12.0
```
Comment (KyleGoyette) [inline charts/launch-agent/Chart.yaml:6]: What are the rules for minor vs patch update here?

**Review by KyleGoyette — COMMENTED** [review]
Couple of qs

**Review by KyleGoyette — APPROVED** [review]
Stamped since testing with an empty object worked

File: charts/launch-agent/README.md (hunk 1 above)
Comment (bcsherma) [reply charts/launch-agent/README.md:49]: nope 👍🏻

File: charts/launch-agent/Chart.yaml (hunk 2 above)
Comment (bcsherma) [reply charts/launch-agent/Chart.yaml:6]: arbitrary. I'm actually going to do this as a patch

**Pull request merged**